### ISSUE TITLE
Generated word document is not displaying correctly

### ISSUE THREAD

**Issue by tdonovancsra** [Alfresco/alfresco-ng2-components#1881]
**Type of issue:**  (check with "[x]")
```
- [ ] New feature request
- [X ] Bug  
- [ ] Support request
- [ ] Documentation
```
**Current behaviour:**
When a form is displaying a generated word document the word document shows up correctly in the AngularJS app fine but in ADF it shows as [object Object].

**Expected behavior:**
Show up in some approximation of what the Angular 1 site does

**Steps to reproduce the issue:**
Generate a word document
display that document in a display value field

**Component name and version:** 
task-details component

**Browser and version:** 
Chrome latest

**Node version (for build issues):**  
6.9.5

**Comment by tdonovancsra** [comment]
+1 Can someone check on this as soon as possible?

**Comment by DenisVuyka** [comment]
This issue is fixed and should get into the upcoming 1.5.0 release.
